index.js

```diff
@@ -25,16 +26,35 @@ var getObject = function (detail) {
 	}
 	return obj;
 };
+var getBorderChars = function (borderStyle) {
+	var chars;
+	if(typeof borderStyle === 'string') {
+		chars = borderChars[borderStyle] || null;
+	} else {
+		var neededKeys = ['tl', 'tr', 'br', 'bl', 'vertical', 'horizontal'];
```

Comment:
I would do it like this

```js
['tl', 'tr', 'br', 'bl', 'vertical', 'horizontal'].forEach(function (key) {
    if(!borderStyle[key] || typeof borderStyle[key] !== 'string') {
        throw new Error('Invalid borderstyle');
    } 
});
```

It's much shorter and you can remove some other stuff. Not sure about the error though. Should it be a `TypeError`? What about the messsage?